src/router/index.ts

```diff
@@ -42,6 +42,7 @@ const router = new VueRouter({
   routes,
 });
 router.beforeResolve((to, from, next) => {
+  store.dispatch(`pegInTx/${constants.PEGIN_TX_ADD_CURRENT_VIEW}`, to.name);
```

Comment:
I think this should not be part of peginTx.
We will most likely use this for other areas in the future (e.g. pegout status, pegout creation...)

I propose creating another entry in the store... `view/` maybe? and the action should probably be `SET_CURRENT_VIEW` rather than `ADD_`